lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1266,7 +1267,7 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     // this number output slices each iteration.
     unsigned unrollDFactor = 1;
 
-    if ((destDepth % 8) == 0) {
+    if ((destDepth % 8) == 0 && (destDepth / 8) % (CI->getGroup()) == 0) {
```

Comment:
Please use `group` variable defined above.

Reply:
group is not a "size_t" type. It is lvm::value